implementations/rust/ockam/ockam_command/tests/commands.bats

```diff
@@ -190,6 +190,43 @@ teardown() {
   assert_failure
 }
 
+@test "vault delete" {
```

Comment:
I'd rather rename this to `vault CRUD` or something like that and remove the `vault create` test.